Automattic/crowdsignal-ui
# Update TextQuestion block UI

This patch updates the UI for the `TextQuestion` block (renamed from `FreeText`) to match the designs in both the editor and on the preview.

The block now supports color and border settings, a resizable textarea and a toggle for making it mandatory and causing an `*` to appear next to the question.

![Screen Shot 2021-10-14 at 10 07 09 PM](https://user-images.githubusercontent.com/8056203/137390924-992a0fc8-5d9c-47fd-9d5d-da38d6a49c7f.png)

_Note: due to the name being changed, any free text blocks will no longer load. We'll also need to update the backend to match the updated label._

Solves c/UdBHjBm2-tr.

# Testing

- Go to `/project` to start a new project.
- Add a `TextQuestion` block. 
- Proceed to make changes to the block as well as its settings accessible through the sidebar.
- Save the project and go to `/project/:projectId/preview`.
- Verify the block looks exactly as it did in the editor.
